package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "leaflet-tilelayer-geojson",
+  "version": "1.0.4",
```

Comment:
version `1.0.4` doesn't exist. I only have a tag for `1.0` (https://github.com/glenrobertson/leaflet-tilelayer-geojson/releases/tag/v1.0)

can you please change it to 1.0?